### PR TITLE
Rework comment layout

### PR DESCRIPTION
**Pull request by HelloWorld017**

### CODE REVIEW

File: src/components/PostComment.vue [hunk 1]
```diff
@@ -1,134 +1,123 @@
...
-                @click="deleteComment"
-                class="dropdown-item"
-              >
-                {{ $t('delete') }}
-              </a>
-              <a v-else href="#" class="dropdown-item">
-                {{ $t('report') }}
-              </a>
+  <div class="comment-wrapper">
+    <div class="comment" :class="{'comment--recomment': isRecomment}" v-if="!isEditing">
```
Comment (litmify) [inline src/components/PostComment.vue:3]: I think _replyComment(or reply-comment or others)_ is easier to understand than _recomment_.

File: src/components/PostComment.vue [hunk 2]
```diff
@@ -1,134 +1,123 @@
...
+
+      <div v-show="showRecommentInput">
+        <PostCommentEditor
+          :parent-comment="comment.id"
+          ref="commentEditor"
+          @upload="$emit('upload', $event)"
+          @close="showRecommentInput = false"
+        />
       </div>
     </div>
```
Comment (litmify) [inline src/components/PostComment.vue:86]: Same as above. Please change all '_recomment_' words below if you rename it.

File: src/views/Post.vue [hunk 3]
```diff
@@ -50,6 +54,7 @@ export default {
       /* Then fetch data from DB. */
       this.refresh()
     },
+
     // @TODO: 매번 refresh 하는게 최선인지는 좀 생각해 봐야할듯
     async refresh () {
       this.post = await fetchPost({ postId: this.postId })
```
Comment (litmify) [inline src/views/Post.vue:60]: Does comment write/update/delete API returns updated comments array or object? If it does, please update the comment object(or an array) with that return value.

Or if retrieve comments API is available, call it and update the comment object.

Comment (HelloWorld017) [reply src/views/Post.vue:60]: Oh I think I've forgot updating it. Thank you for pointing out!

**Pull request merged**